index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="amen">
+    <header class="navItems">
+     
+        
+        <div class="images">
+         <div class="space">
+            <div class="image01"><img src="assets/Shape.png" alt="#" class="shape00"></div>
+            <div class="image02"><img src="assets/Shape_1.png" alt="#"  class="shape01"></div>
+            </div>
+            <div class="menuitem">
+             <div class="boxic"><img src="assets/ic_menu.svg"  alt="#" class="ic_menu"></div>  
```

Comment:
- [ ] [REQUIRED] As requested by the previous reviewer, please make this a background image in CSS as it is decoration. This is a good practise and it serves many advantages even in manipulating different effects to it.